integration-tests/typescript-fetch/src/uniform-github-repositories/uniform-github-repositories.ts

```diff
@@ -22,6 +26,15 @@ async function main() {
     console.info("updating: " + repo.full_name)
     await updateRepoConfig(repo.owner.login, repo.name)
   }
+
+  if(Reflect.has(global, 'gc')){
+    // @ts-ignore
+    global.gc()
+  }
+
+  setTimeout(() => {
+    writeHeapSnapshot()
```

Comment:
Analyzed heap snapshots to make sure that the `AbortController` / `AbortSignal` aren't being accidentally retained due to the subscription to the `AbortSignal`s `abort` event as implied by https://github.com/elastic/kibana/issues/65051 - as far as I can tell they are not